src/components/form/Select.js

```diff
@@ -9,57 +8,47 @@ import { formInput, formOptions } from '../../utils/react/propTypes'
 import { styles } from './styles'
 import { ErrorText } from './buildingBlocks/ErrorText'
 
-export const Select = withStyles(styles)(props => (
-    <div className={props.classes.container}>
-        <Field name={props.name} placeholder={props.placeholder}>
-            {({ input, meta, placeholder }) => (
-                <StyledSelect
-                    placeholder={placeholder}
-                    input={input}
-                    showEmptyOption={props.showEmptyOption}
-                    showErrorText={props.showErrorText}
-                    options={props.options}
-                    error={meta.error || ''}
-                    touched={meta.touched}
-                    classes={props.classes}
-                />
-            )}
-        </Field>
-    </div>
-))
+export const SelectField = props => (
+    <MUISelect native {...props.input}>
+        {props.showEmptyOption && <option value="" />}
+        {props.options.map(option => (
+            <option key={option.value} value={option.value}>
+                {option.label}
+            </option>
+        ))}
+    </MUISelect>
+)
 
-Select.propTypes = {
-    name: PropTypes.string.isRequired,
+SelectField.propTypes = {
+    input: formInput.isRequired,
+    showEmptyOption: PropTypes.bool.isRequired,
     options: formOptions.isRequired,
-    placeholder: PropTypes.string.isRequired,
-    showEmptyOption: PropTypes.bool,
-    showErrorText: PropTypes.bool,
-}
-
-Select.defaultProps = {
-    showEmptyOption: false,
-    showErrorText: false,
 }
 
-const StyledSelect = props => (
-    <FormControl className={props.classes.formControl}>
-        <InputLabel htmlFor={props.input.name}>{props.placeholder}</InputLabel>
-        <SelectField
-            input={props.input}
-            showEmptyOption={props.showEmptyOption}
-            options={props.options}
-        />
-        <ErrorText
-            showErrorText={props.showErrorText}
-            error={props.error}
-            touched={props.touched}
-        />
-    </FormControl>
+export const Select = props => (
+    <div className={props.classes.container}>
+        <FormControl className={props.classes.formControl}>
+            <InputLabel htmlFor={props.input.name}>
+                {props.placeholder}
+            </InputLabel>
+            <SelectField
+                input={props.input}
+                showEmptyOption={props.showEmptyOption}
+                options={props.options}
+            />
+            <ErrorText
+                showErrorText={props.showErrorText}
+                error={props.error}
+                touched={props.touched}
+            />
+        </FormControl>
+    </div>
 )
 
-StyledSelect.propTypes = {
+Select.propTypes = {
     classes: PropTypes.shape({
```

Comment:
At first I added those as well, but then I realised that withStyles is like a HOC that will handle all the stuff, so I thought I just leave it out. There weren't any linting errors either.
I think this should go out, but just a preference, not a requested change

Reply:
I'm fine leaving them out. I think my linter complained, that's why I added em.